runner/pool/pool.go

```diff
@@ -394,7 +394,11 @@ func (r *basePoolManager) AddRunner(ctx context.Context, poolID string) error {
 		return errors.Wrap(err, "fetching pool")
 	}
 
-	name := fmt.Sprintf("garm-%s", uuid.New())
+	prefix := pool.RunnerPrefix
+	if prefix == "" {
+		prefix = "garm"
```

Comment:
Let's define a constant or a variable ```DefaultRunnerPrefix```. We may need to use it in more than one place and will make our lives easier if we decide to use another default.

Reply:
done